app/components/works/edit/file_row_component.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module Works
+  module Edit
+    # Renders a widget corresponding to a single file attached to the work.
+    class FileRowComponent < ApplicationComponent
+      def initialize(form:, zip_template: false)
+        @form = form
+        @zip_template = zip_template
+      end
+
+      attr_reader :form
+
+      def filename
+        return unless uploaded?
+
+        form.object.model.path
```

Comment:
Lines are uncovered. Consider tossing a component test at this or is it more trouble than it's worth?

Reply:
good find - this actually appears to be a dead code path -- removing from here and the view